src/pages/Search/useHandleBackButton/index.android.ts

```diff
@@ -0,0 +1,11 @@
+import {useEffect} from 'react';
+import {BackHandler} from 'react-native';
+import type UseHandleBackButtonCallback from './type';
+
+export default function useHandleBackButton(callback: UseHandleBackButtonCallback) {
```

Comment:
Can we instead extend the logic in [src/libs/Navigation/setupCustomAndroidBackHandler](https://github.com/Expensify/App/tree/76c3a616062d0ba04c90da085a9ab598c69aaef5/src/libs/Navigation/setupCustomAndroidBackHandler)?